servlet-feature-pack/src/main/resources/content/welcome-content/documentation.html

```diff
@@ -64,8 +64,8 @@
 
         <p class="logos"><a href="http://jboss.org"><img src="jbosscommunity_logo_hori_white.png" alt="JBoss and JBoss Community" width="195" height="37" border="0"></a></p>
 
-        <p class="note">To replace this page set "enable-welcome-root" to false in your server configuration and deploy
-            your own war with / as its context path.</p>
+        <p class="note">To replace this page simply deploy your own war with / as its context path.<br/>
+            To disable it, remove "welcome-content" handler for location / in undertow subsystem.</p>
```

Comment:
Haha, nice one, thanks. Looks like they are saving on definite articles in Australia! 

Fixed in all 3 places.